titus-server-runtime/src/main/java/com/netflix/titus/runtime/KubeSchedulerFeatureConfiguration.java

```diff
@@ -40,4 +40,10 @@
      */
     @DefaultValue(".*")
     String getEnabledMachineTypes();
+
+    /**
+     * Set to true to enable routing jobs with EBS volume configuration to KubeScheduler.
+     */
+    @DefaultValue("false")
```

Comment:
Will there be a corresponding nftitus PR to enable this or will this just be done via FP?

Reply:
This property should be sufficient.